p2p-specs/p2p-interface.md

```diff
@@ -168,22 +173,28 @@ For any optional queueing, Bundlers SHOULD maintain maximum queue sizes to avoid
 
 #### Global topics
 
-The primary global topics used to propagate user operations to all nodes on the network is `UserOperationsWithEntryPoint`.
+The primary global topics used to propagate user operations to all nodes on the network is `verified_user_operations`.
+
+##### `verified_user_operations`
+
+The `verified_user_operations` topic is used to gossip user operations on mempool topics where the user operation is valid.
```

Comment:
not sure why we specify "verified" as part of the protocol topic name. do we imply there are situations we have a protocol for unverified data?